src/runtime/markdown-parser/handler/link.ts

```diff
@@ -25,3 +26,11 @@ export default function link (h: H, node: Node) {
 
   return h(node, 'a', props, all(h, node))
 }
+
+function normalizeLink (link: string) {
+  if (isRelative(link)) {
+    return link.replace(/^[0-9]*\./g, '').replace(/\.md$/g, '')
```

Comment:
```suggestion
    return link.replace(/^(.\/)?[0-9]*\./g, '$1').replace(/\.md$/g, '')
```

Reply:
It forgets `../` situation.

Reply:
True, we should also think about multipart links

`./1.test/2.index.md`